scripts/pyhelpers/README.md

```diff
@@ -0,0 +1,3 @@
+# pyhelpers
```

Comment:
I feel like the record-and-replay tooling deserves its own proper home in the dir structure, instead of `scripts/`.

Reply:
> I feel like the record-and-replay tooling deserves its own proper home in the dir structure, instead of `scripts/`.

I'd like to keep things as light weight as possible, but in the future if the tooling starts to get more complex, we can move it to a separate project dir (with fully fleshed out pipenv setup and everything :) ). I think this is appropriate for now though